recipe/meta.yaml

```diff
@@ -29,20 +27,23 @@ build:
 
```

Comment:
Add  `skip: True   # [win and vc!=9]`

Reply:
Also
```
track_features:
- vc9               # [win]
```

can be removed I think. We removed it for other python versions

Reply:
Thanks! I had those when I copied the recipe from AnacondaRecipes but ended up reverting and doing a bad copy-n-pasta.